tests/KdybyTests/Translation/LatteExtractorTest.phpt

```diff
@@ -43,17 +58,34 @@ class LatteExtractorTest extends \KdybyTests\Translation\TestCase
 
 		$catalogue = new MessageCatalogue('cs_CZ');
 		$extractor->extract(__DIR__ . '/data/extractor-files', $catalogue);
-
-		Assert::same([
-			'messages' => [
-				'homepage.Important title' => 'Important title',
-				'homepage.Another important title' => 'Another important title',
-				"homepage.\nInteresting article about interesting topic\n" => "\nInteresting article about interesting topic\n",
-				'homepage.Chapter 2' => 'Chapter 2',
-				'homepage.none|one|many' => 'none|one|many',
-				'homepage.sample.identificator' => 'sample.identificator',
-			],
-		], $catalogue->all());
+		if (!defined(Engine::class . '::VERSION_ID') || Engine::VERSION_ID < 20900) {
+			$mess = [
+				'messages' => [
+					'homepage.Important title' => 'Important title',
+					'homepage.Another important title' => 'Another important title',
+					"homepage.\nInteresting article about interesting topic\n" => "\nInteresting article about interesting topic\n",
+					'homepage.Chapter 2' => 'Chapter 2',
+					'homepage.none|one|many' => 'none|one|many',
+					'homepage.sample.identificator' => 'sample.identificator',
+				],
+			];
+		} else {
+			$impT = sprintf('%s%s%s', '(', '"Important title"', ')');
+			$prefixImpT = sprintf('%s%s%s', 'homepage.(', '"Important title"', ')');
+			$prefixNoneOneMany = sprintf('%s%s', 'homepage.', 'none|one|many');
+			$prefixSampleIdentificator = sprintf('%s%s', 'homepage.sample.', 'identificator');
```

Comment:
These variables seem useless, the result is the same as in the 2.9 case.

Reply:
I agree. It is fixed.